src/Jackett.Common/Definitions/thepiratebay.yml

```diff
@@ -7,8 +7,6 @@
   encoding: UTF-8
   links:
     - https://thepiratebay.org/
-    - https://thepiratesbay.pw/
```

Comment:
instead of removing the links, please add them back in with a   legacylinks: header, this way users who may still have those old links active will be auto-converted to the valid sitelink.
```yaml
  legacylinks:
    - https://thepiratesbay.pw/
    - https://tproxy.pro/
```

Reply:
ah, understood! I wasn't aware of that functionality. new commit is pushed. thanks!